pkg/icingaredis/heartbeat.go

```diff
@@ -94,14 +94,11 @@ func (h *Heartbeat) controller(ctx context.Context) {
 		defer throttle.Stop()
 
 		for {
-			cmd := h.client.XRead(ctx, &redis.XReadArgs{
+			streams, err := h.client.XReadUntilResult(ctx, &redis.XReadArgs{
 				Streams: []string{"icinga:stats", "$"},
-				Block:   0, // TODO(el): Might make sense to use a non-blocking variant here
 			})
```

Comment:
Do we really want that special-handling of `$` in `XReadUntilResult()` here? Using `$` in the old code ensured that when a message is returned by Redis, it was just written and `time.Now()` is a good estimate to when this heartbeat was written. This is no longer the case now and the heartbeat could be a few minutes old.